docs/docs/09.3-environments.md

```diff
@@ -29,6 +29,11 @@ If you use JSX, you may want to pre-compile your components. Alternatively you m
 >
 > Some versions of Node.js have an `Object.assign` implementation that does not preserve key order. This can cause errors when validating the markup, creating a warning that says "React attempted to reuse markup in a container but the checksum was invalid". If you run into this issue, you can override `Object.assign` to use a polyfill that preserves key order. For more details, see [Issue #6451](https://github.com/facebook/react/issues/6451).
 
+C#
```

Comment:
Should this be .NET? I would expect this to work with any other .NET language but I might be wrong.